### ISSUE TITLE
Commands for statistics to help players understand themselves and the meta

### ISSUE THREAD

**Issue by BadWolf1023** [BadWolf1023/MKW-Table-Bot#31]
After #28 is finished, commands should be implemented that help players understand their best tracks, their worst tracks, "hot" tracks, and "underplayed" tracks. Details:

If *days* are not provided, it will go by your entire history. Otherwise, it will only consider that last *x* days.
If *tier* is not provided, it will go by all tiers. Otherwise, it will only consider matches you played in a specific tier.

**These commands are not finished yet:**
- `?besttracks ct (tier) (days)` - displays in order your top 20 CT tracks based on how how many seconds behind first place you usually finish (or perhaps how many seconds ahead of the average you finish). 
- `?worsttracks ct (tier) (days)` - displays in order your worst 20 CT tracks based on how how many seconds behind first place you usually finish (or perhaps how many seconds behind the average you finish).
- `?topplayers track (tier) (days)` - displays in order the top 20 players for a given track based on how how many seconds behind first place each player finishes (or perhaps how many seconds ahead of the average the player finishes). A user specified track should not need to match a track name exactly. Instead, Table Bot should choose the track that best matches the input for the given track. This could be as simple as a substring match (such as "Gorge" finds the top players on Mushroom Gorge), but could employ more advanced techniques such as abbreviations (such as "SKL" For example), and could even further employ advanced information retrieval techniques such as stopword removal, stemming, index term weighting, etc (but these may be out of scope and likely not needed). At a minimum, substring matching should be implemented.

**These commands are already finished:**
- `?hottracks ct (tier) (days)` - displays in order the top 20 played CTs.
- `?underplayedtracks ct (tier) (days)` - displays in order the 20 CTs that are least played.
